### PR TITLE
ci: Add GitHub Actions to build OpenXR-CTS for Android

### PR DESCRIPTION
**Pull request by utzcoz**

### CODE REVIEW

**Comment by utzcoz** [comment]
Looks like the PR that adds GitHub Workflow firstly will not run.

File: .github/workflows/android-OpenXR-CTS-ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# SPDX-License-Identifier: CC0-1.0
+
+name: Android OpenXR-CTS - Check
+on:
+  pull_request:
+    branches:
+      - 'devel'
+
+jobs:
+  build-helloxr:
```
Comment (rpavlik) [inline .github/workflows/android-OpenXR-CTS-ci.yml]: Probably change the job name :grin:

**Review by rpavlik — CHANGES_REQUESTED** [review]
Looks good, I presume it's based on my workflow for hello_xr. Just one little tweak and we can get this merged. Thanks so much!

File: .github/workflows/android-OpenXR-CTS-ci.yml (hunk 1 above)
Comment (utzcoz) [reply .github/workflows/android-OpenXR-CTS-ci.yml]: Done. Thanks.

**Comment by utzcoz** [comment]
> Looks good, I presume it's based on my workflow for hello_xr. Just one little tweak and we can get this merged. Thanks so much!

Yep :). I copied from OpenXR-SDK-Source, and changed something for OpenXR-CTS.

**Review by rpavlik — APPROVED** [review]
Looks good! I'll merge it after I tag the release I'm finishing up right now

**Review by rpavlik — APPROVED** [review]

**Comment by utzcoz** [comment]
Thanks.

**Comment by rpavlik** [comment]
Thank you!

**Pull request merged**